Sources/BraintreePayPal/RecurringBillingMetadata/BTAmountBreakdown.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+/// A recurring billing amount breakdown.
+///
```

Comment:
nit: remove extra comment line
```suggestion
///
```

Reply:
I had suggested this in an earlier PR to improve readability. :)